.github/workflows/test-other.yaml

```diff
@@ -0,0 +1,48 @@
+# Other tests, primarily those requiring obsolete dependencies.
+name: test-other
+
+on:
+  push:
+    branches:
+      - "*"
+
+jobs:
+  run-tox:
+    name: tox -e ${{ matrix.toxenv }}
+    runs-on: ubuntu-20.04
```

Comment:
can't we use a more modern version? like 22 or 24?

Mainly because I don't know how long will github keep 20 around, since in 6 month id will be unsupported

Reply:
I can check, but unfortunately I don't think so. We left on 20.04 on purpose when were were upgrading to 22.04.
In a pinch, you can probably compile from source...

Reply:
Hm, `libunwind-dev` itself seems to work just fine. :thinking: 
I can try a combo of Ubuntu 22.04 and old Python.

Reply:
I'll merge this one, and try to experiment in a new PR.

Reply:
my suggestion is just to having having to do stuff in 6 month when we don't remember what this is doing

Reply:
That might make sense :smiling_imp: But we can start to demoting stuff to only be run on master merge.